src/enum_table.cpp

```diff
@@ -171,6 +171,12 @@ const struct EnumItem ENUM_SystemViewMode[] = {
 	{ 0, 0 },
 };
 
+const struct EnumItem ENUM_SystemSelectionMode[] = {
+	{ "SELECTED_SYSTEM", int(SystemView::SystemSelectionMode::SelectedSystem) },
+	{ "CURRENT_SYSTEM", int(SystemView::SystemSelectionMode::CurrentSystem) },
+	{ 0, 0 },
+};
+
```

Comment:
Note that these are generated by running `scripts/scan_enums.py -o src/enum_table.cpp --pattern='*.h' -r src/`. The script reads appropriately-annotated C++ `enum` declarations and generates the appropriate entries. Manual changes to these files will be overridden.

Reply:
Fixed - please see https://github.com/pioneerspacesim/pioneer/pull/5922#discussion_r1789093383 for details.